src-vue/src/i18n/lang/fr.ts

```diff
@@ -0,0 +1,148 @@
+export default {
+    menu: {
```

Comment:
What happens if a language is missing a key. Does it fall back to English?

Reply:
Yep, it defaults to English, as set in `main.ts`:
```javascript
export const i18n = createI18n({
    locale: 'en',
    fallbackLocale: 'en',
    messages: {
        en, fr
    }
});
```